admin/roles.sql

```diff
@@ -40,3 +40,6 @@ CREATE ROLE IF NOT EXISTS SYNAPSE_DATA_WAREHOUSE_ANALYST;
 CREATE ROLE IF NOT EXISTS SYNAPSE_DATA_WAREHOUSE_DEV_ADMIN;
 CREATE ROLE IF NOT EXISTS SYNAPSE_DATA_WAREHOUSE_DEV_ANALYST;
 
+-- Data Warehouse proxy admin roles
+CREATE ROLE IF NOT EXISTS SYNAPSE_DATA_WAREHOUSE_ALL_ADMIN;
+CREATE ROLE IF NOT EXISTS SYNAPSE_DATA_WAREHOUSE_DEV_ALL_ADMIN;
```

Comment:
My apologies because this changes a lot of things, but the name `ALL_ADMIN` is confusing with `ADMIN`. Can we change the name to `PROXY_ADMIN`?

Reply:
Sure.